src/components/Heading/Heading.js

```diff
@@ -8,17 +8,33 @@ import { sizes } from '../../styles/constants';
 
 const { KILO, MEGA, GIGA, TERA, PETA, EXA, ZETTA } = sizes;
 
+const mobileSizeMap = {
+  [KILO]: KILO,
+  [MEGA]: MEGA,
+  [GIGA]: MEGA,
+  [TERA]: GIGA,
+  [PETA]: TERA,
+  [EXA]: PETA,
+  [ZETTA]: PETA
+};
+
 const baseStyles = ({ theme }) => css`
   label: heading;
   font-weight: ${theme.fontWeight.bold};
   margin-bottom: ${theme.spacings.giga};
 `;
 
-const sizeStyles = ({ theme, size }) => css`
-  label: heading--${size};
-  font-size: ${theme.typography.headings[size].fontSize};
-  line-height: ${theme.typography.headings[size].lineHeight};
-`;
+const sizeStyles = ({ theme, size }) =>
+  css`
+    label: heading--${size};
+    font-size: ${theme.typography.headings[mobileSizeMap[size]].fontSize};
```

Comment:
These are getting a bit long 😂 

Minor suggestion, would it make sense to have a function for this? I imagine something where only the sizes that need to get adjusted are inside the map and everything else is looked up as is.

``` javascript
const mapMobileSize = theme => size => {
  const mobileSizeMap = {};

  const mappedSize = mobileSizeMap[size] || size;

  return theme.typography.headings[mappedSize];
};
```

And then you can obtain font size and line height from there? Makes those object accesses in your styles smaller, but also requires `sizeStyles` to not be immediately returning.

Just a suggestion, feel free to ignore it.